blog/urls.py

```diff
@@ -0,0 +1,14 @@
+from blog.models import Commentary
+from blog.views import index, PostDetailView, CommentaryCreateView
+
+from django.urls import path
+
+app_name = "blog"
+
+urlpatterns = [
+    path("", index, name="index"),
+    path("posts/<int:id>/", PostDetailView.as_view(), name="post-detail"),
+    path("posts/<int:id>/create/",
+         CommentaryCreateView.as_view(),
+         name="create"),
```

Comment:
The URL pattern for creating a commentary might be incorrect. Typically, the creation of a commentary would be associated with a specific post, so the URL might need to reflect that relationship more clearly. Consider reviewing the URL structure to ensure it aligns with the intended functionality.